applications/views.py

```diff
@@ -265,3 +268,19 @@ def post(self, request, *args, **kwargs):
             )
 
         return Response(data={"interview_link": interview_link})
+
+
+class LettersView(TemplateView):
+    """
+    Render a letter sent to an applicant
+    """
+
+    template_name = "letter_template_page.html"
+
+    def get_context_data(self, **kwargs):
+        """
+        Fill in variables in the letter template
+        """
+        hash_code = kwargs.get("hash")
+        letter = get_object_or_404(ApplicantLetter, hash=hash_code)
+        return {**get_base_context(self.request), "content": letter.letter_text}
```

Comment:
Seems to me like we should be passing the signatory name and the signature image path/alt text in this context dict, and not hardcoding those things directly into the template. We should be able to add those as properties to `LetterTemplatePage`, right?

Reply:
I discussed with @pdpinch and we are fine hardcoding it for now. I think we need to do more thinking around customizing this before moving forward.